### PR TITLE
Replace 9.2 by 10.0

### PR DESCRIPTION
**Pull request by ho4ho**
9.2 should 10.0

### CODE REVIEW

**Comment by settermjd** [comment]
Thanks for contributing this, @ho4ho.

File: modules/admin_manual/pages/maintenance/manual_upgrade.adoc [hunk 1]
```diff
@@ -214,7 +214,7 @@ Then the copied tables were deleted after the upgrade was completed.
 This doubled the upgrade time, so admins could skip this test (by
 risking a failed upgrade) with `php occ upgrade --skip-migration-test`.
 
-The migration test has been removed from ownCloud 9.2. ownCloud server
+The migration test has been removed from ownCloud 10.0. ownCloud server
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/manual_upgrade.adoc]: As the original change was referring specifically to ownCloud 9.2, I wouldn't update the version number. Instead, I'd change the sentence to: "The migration test has been removed."

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ See xref:upgrading-across-skipped-releases[Upgrading Across Skipped Releases] fo
 * xref:installation/apps_management_installation[Disable] all third-party apps.
 * Make xref:maintenance/backup.adoc[a fresh backup].
 * Upgrade your ownCloud packages.
-* Run xref:configuration/server/occ_command.adoc:command-line-upgrade[occ upgrade] (The optional parameter to skip migration tests was removed from oC 9.2. See xref:maintenance/upgrade.adoc[Testing a Migration] for background information).
+* Run xref:configuration/server/occ_command.adoc:command-line-upgrade[occ upgrade] (The optional parameter to skip migration tests was removed from ownCloud 10.0. See xref:maintenance/upgrade.adoc[Testing a Migration] for background information).
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc]: See above.

File: modules/admin_manual/pages/maintenance/package_upgrade.adoc [hunk 3]
```diff
@@ -82,7 +82,7 @@ This example is for CentOS/RHEL/Fedora:
 sudo -u apache php occ upgrade
 ....
 
-The optional parameter to skip migration tests during this step was removed in oC 9.2.
+The optional parameter to skip migration tests during this step was removed in ownCloud 10.0.
```
Comment (settermjd) [inline modules/admin_manual/pages/maintenance/package_upgrade.adoc]: See above.

**Comment by ho4ho** [comment]
Remove completely instead as already noticed in https://github.com/owncloud/docs/blob/master/modules/admin_manual/pages/release_notes.adoc#changes-in-1001 ?

**Comment by settermjd** [comment]
@ho4ho, no need to close the PR. It's small but still valuable.

**Comment by settermjd** [comment]
Closed by accident.

**Comment by settermjd** [comment]
Thanks for your time in creating this PR, @ho4ho. Feel free to create future PRs, as and when you see the need to do so.

**Comment by ho4ho** [comment]
@settermjd Why replace 10.0 by 9.2 again with:

https://github.com/owncloud/docs/pull/804/commits/3eab15739e3b29d73548c55e579fbda1e50e80fc 
https://github.com/owncloud/docs/pull/804/commits/3deb10d074d13e1adb6ecab638c89cc21bcaf9eb
https://github.com/owncloud/docs/pull/804/commits/52e6eebfe6b0894b76d116fc6323b47470a55274
https://github.com/owncloud/docs/pull/804/commits/db240ebad0ff81f09f2d301879dde710a5c7a3e1

?

**Comment by settermjd** [comment]
With respect to https://github.com/owncloud/docs/pull/804#issuecomment-481723006, because the changes weren't made in 10.0. I should have picked this up at the time, however.

**Comment by ho4ho** [comment]
But 10.0 is 9.2 where this get removed: https://github.com/owncloud/core/pull/26172

**Pull request merged**